src/Workleap.OpenApi.MSBuild/OasdiffManager.cs

```diff
@@ -43,16 +43,18 @@ public async Task RunOasdiffAsync(IEnumerable<string> openApiSpecFiles, IEnumera
 
         foreach (var baseSpecFile in openApiSpecFiles)
         {
-            this._loggerWrapper.LogMessage($"\n ******** Oasdiff: Diff comparison with {baseSpecFile} ********", MessageImportance.High);
-            
             var fileName = Path.GetFileName(baseSpecFile);
-            var generatedSpecFilePath = generatedOpenApiSpecFilesList.FirstOrDefault(x => x.Contains(fileName));
+            this._loggerWrapper.LogMessage($"\n ******** Oasdiff: Diff comparison with {fileName} ******** \n", MessageImportance.High);
             
+            var generatedSpecFilePath = generatedOpenApiSpecFilesList.FirstOrDefault(x => x.Contains(fileName));
```

Comment:
I know this isn't your code, but I wish we did better file name comparison here. I don't believe that checking if a file name is somewhere in a path is OK

Reply:
updated!